docs/design/developer_notes/PCSOperations.txt

```diff
@@ -33,10 +33,10 @@ POST /transitions
    o Create reservations for targs that need them.  Go lib keeps these alive.
      Reservations are best-effort -- not "all or nothing".
 
-   o Get the HW state of the targets.  Use ETCD HW state records, or
-     go to the HW and ask (TBD).  If any targets are already at the destination
-     state, remove these from the list and mark the return data for that target
-     as 'compete', or maybe 'already there' in taskStateDescription.
+   o Get the HW state of the targets. Use ETCD HW state records. If any targets
+     are already at the destination state, remove these from the list and mark
+     the return data for that target as 'compete', or maybe 'already there' in
```

Comment:
super duper minor nit since this is just dev notes: s/compete/complete

Reply:
I'll make this change before merging

Reply:
Done